packages/http/libraries/validation/zod/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "author": "Adrián Martínez Jiménez",
+  "bugs": {
+    "url": "https://github.com/inversify/monorepo/issues"
+  },
+  "description": "InversifyJs http zod validation package",
+  "dependencies": {
+    "@inversifyjs/http-core": "workspace:*",
+    "@inversifyjs/reflect-metadata-utils": "workspace:*"
+  },
+  "devDependencies": {
+    "@inversifyjs/http-express": "workspace:*",
+    "@stryker-mutator/core": "9.0.1",
+    "@stryker-mutator/typescript-checker": "9.0.1",
+    "@stryker-mutator/vitest-runner": "9.0.1",
+    "@types/express": "5.0.3",
+    "@types/node": "22.16.5",
+    "@vitest/coverage-v8": "3.2.4",
+    "eslint": "9.31.0",
+    "prettier": "3.6.2",
+    "rimraf": "6.0.1",
+    "rollup": "4.45.1",
+    "ts-loader": "9.5.2",
+    "tslib": "2.8.1",
+    "typescript": "5.8.3",
+    "vitest": "3.2.4"
+  },
+  "devEngines": {
+    "node": "^22.10.2",
+    "pnpm": "^9.12.1"
+  },
+  "homepage": "https://inversify.io",
+  "keywords": [
+    "dependency injection",
+    "dependency inversion",
+    "di",
+    "inversion of control container",
+    "ioc",
+    "javascript",
+    "node",
+    "typescript"
+  ],
+  "license": "MIT",
+  "main": "lib/cjs/index.js",
+  "module": "lib/esm/index.js",
+  "exports": {
+    ".": {
+      "import": "./lib/esm/index.js",
+      "require": "./lib/cjs/index.js"
+    }
+  },
+  "name": "@inversifyjs/zod-validation",
```

Comment:
I'm not sure if that's the most appropriate name. Ultimately, this package has to be used in the context of http or it won't work. Wouldn't it be better to name it in that context?

Reply:
Absolutely, you're right